website/content/docs/k8s/multiport/configure.mdx

```diff
@@ -0,0 +1,478 @@
+
+The following examples create Consul CRDs that allow traffic to only one port of the multi-port service. Each resource separately denies `web` permission when it is a source of traffic to one of the services. These traffic permissions work with either method for defining a multi-port service.
+
+<CodeTabs tabs={[ "Deny port 80", "Deny port 90" ]}>
+
+<CodeBlockConfig filename="deny-80.yaml">
+
+```yaml
+apiVersion: auth.consul.hashicorp.com/v2beta1
+kind: TrafficPermissions
+metadata:
+  name: web-to-api-admin-port-allow
+spec:
+  destination:
+    identityName: api
+  action: deny
+  permissions:
+    - sources:
+        - identityName: web
+      destinationRules:
+        - portNames: ["api"]
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig filename="deny-90.yaml">
+
+```yaml
+apiVersion: auth.consul.hashicorp.com/v2beta1
+kind: TrafficPermissions
+metadata:
+  name: web-to-api-port-allow
+spec:
+  destination:
+    identityName: api
+  action: deny
+  permissions:
+    - sources:
+        - identityName: web
+      destinationRules:
+        - portNames: ["admin"] 
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+## Validate multi-port connection
+
+To open a shell to the `web` container, you need the name of the pod it currently runs on. Run the following command to return a list of pods:
+
+```shell-session
+$ kubectl get pods
+NAME                   READY   STATUS    RESTARTS   AGE
+api-7dc9d84f-kfnwz     2/2     Running   0          23s
+web-7bc5786747-b5pzl   1/1     Running   0          23s
+```
+
+### Validate both ports
+
+Use the `web` Pod's name to open a shell session and test the `api` service.
+
+<Tabs>
+
+<Tab heading="Method 1" group="method1">
+
+```shell-session
+$ kubectl exec -it web-7bc5786747-b5pzl -- curl api:80
+hello world
+```
+
+Then test the `api-admin` service.
+
+```shell-session
+$ kubectl exec -it web-7bc5786747-b5pzl -- curl api:90
+hello world from 9090 admin
+```
+
+</Tab>
+
+<Tab heading="Method 2" group="method2">
+
+```shell-session
+$ kubectl exec -it web-7bc5786747-b5pzl -- curl api:80
+hello world
+```
+
+Then test the `api-admin` service.
+
+```shell-session
+$ kubectl exec -it web-7bc5786747-b5pzl -- curl api-admin:90
+hello world from 9090 admin
+```
+
+</Tab>
+</Tabs>
+
+### Validate port 80
+
+Apply the CRD to allow traffic to port 80 only:
+
+```shell-session
+$ kubectl apply -f deny-90.yaml
+```
+
+<Tabs>
+
+<Tab heading="Method 1" group="method1">
+
+Then, open a shell session in the `web` container and test the `api` service on port 80.
+
+```shell-session
+$ kubectl exec -it web-7bc5786747-b5pzl -- curl api:80
+hello world
+```
+
+Test the `admin` service on port 90. This command should fail, indicating that the traffic permission is in effect.
+
+```shell-session
+$ kubectl exec -it web-7bc5786747-b5pzl -- curl api:90
+```
+
+</Tab>
+
+<Tab heading="Method 2" group="method2">
+
+Then, open a shell session in the `web` container and test the `api` service on port 80.
```

Comment:
```suggestion
Then, open a shell session in the `web` container and test the `api` service on port `80`.
```
I won't change all instances in case you disagree